src/types/fetch.rs

```diff
@@ -1,5 +1,8 @@
 use super::{Flag, Seq, Uid};
 use imap_proto::types::{AttributeValue, Envelope, MessageSection, SectionPath};
+use chrono::{DateTime, FixedOffset};
+
+const DATE_TIME_FORMAT: &str = "%d-%b-%Y %H:%M:%S %z";
```

Comment:
Could you leave a comment pointing to where this format is standardized?

Reply:
It was planned, but I was about to forget it !